frontend/src/utils/verified-models.ts

```diff
@@ -2,10 +2,10 @@
 export const VERIFIED_PROVIDERS = ["openai", "azure", "anthropic"];
 export const VERIFIED_MODELS = [
   "gpt-4o",
-  "claude-3-5-sonnet-20240620",
```

Comment:
You should not remove this.

Also, make sure you install pre-commit hooks and run `make lint` before commit. Your code has some linter error